cmd/libs/go2idl/make-client/generators/imported_types.go

```diff
@@ -0,0 +1,56 @@
+/* Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generators
+
+import "k8s.io/kubernetes/cmd/libs/go2idl/types"
+
+var (
+	// We use these types in the template to trigger the ImportTracker
+	fieldSelector = &types.Type{
+		Name: types.Name{
+			"k8s.io/kubernetes/pkg/fields",
```

Comment:
Please use `Name:` and `Package:` when using struct literals from other packages.